### DISCUSSION TITLE
Issue viewing Endgame Index with Analyst Role

### DISCUSSION THREAD

**Discussion by smorez-code** [Security-Onion-Solutions/securityonion#11086] [Unsupported Versions]
Does anyone have a fix to allow users who belong to the "analyst" role to be able to view the endgame-* index? Currently, only superusers can view the index. 

I've updated the endgame index pattern in the data view to the correct index pattern. Running a distributed deployment and using SecOnion version  2.3.220. 

Thank you

**Comment by lock-wire** [comment]
You should be able to take the following steps:

1. sudo cp /opt/so/saltstack/default/salt/elasticsearch/roles/* /opt/so/saltstack/local/salt/elasticsearch/roles/
2. sudo vi /opt/so/saltstack/local/salt/elasticsearch/roles/analyst.json
3. Add the endgame index to the index permissions
`"names": [`
`         "so-*",`
`         "endgame-*"`
`       ],`
8. sudo so-elasticsearch-restart

You should see the new index permission in Kibana -> Stack Management -> Security -> Roles
